examples/pytorch/model_zoo/classification.py

```diff
@@ -0,0 +1,87 @@
+from dgl.data import Tox21
+from dgl.data.utils import split_dataset
+from dgl import model_zoo
+import torch
+from torch.nn import BCEWithLogitsLoss
+from torch.optim import Adam
+from torch.utils.data import DataLoader
+
+from utils import Meter, EarlyStopping, collate_molgraphs
+
+def main():
+    device = "cuda" if torch.cuda.is_available() else "cpu"
+    batch_size = 128
+    learning_rate = 0.001
+    num_epochs = 100
+
+    # Interchangable to other Dataset
+    dataset = Tox21()
+    atom_data_field = 'h'
+
+
+    trainset, valset, testset = split_dataset(dataset, [0.8, 0.1, 0.1])
+    train_loader = DataLoader(
+        trainset, batch_size=batch_size, collate_fn=collate_molgraphs)
+    val_loader = DataLoader(
+        valset, batch_size=batch_size, collate_fn=collate_molgraphs)
+    test_loader = DataLoader(
+        testset, batch_size=batch_size, collate_fn=collate_molgraphs)
+
+    # Interchangable to other model in model zoo
+    model = model_zoo.chem.GCNClassifier(74, [64, 64], dataset.n_tasks)
+
+    model.to(device)
+    loss_criterion = BCEWithLogitsLoss(pos_weight=torch.tensor(
+        dataset.task_pos_weights).to(device), reduction='none')
+    optimizer = Adam(model.parameters(), lr=learning_rate)
+
+    stopper = EarlyStopping(patience=10)
+
+    for epoch in range(num_epochs):
+        model.train()
+        print('Start training')
+        train_meter = Meter()
+        model.train()
+        for batch_id, batch_data in enumerate(train_loader):
+            smiles, bg, labels, mask = batch_data
+            atom_feats = bg.ndata.pop(atom_data_field)
```

Comment:
We can probably move this line into `collate_molgraphs`.

Reply:
I think it's fine to put it here, making it consistent with `dataset.__getitem__`